.changeset/tall-carrots-know.md

```diff
@@ -1,5 +1,5 @@
 ---
-'@sumup/circuit-ui': minor
+'@sumup/circuit-ui': patch
```

Comment:
Made this a patch as mentioned and tweaked the changeset (we want it succinct, this is basically a changelog entry)